pages/mitarbeiter.js

```diff
@@ -0,0 +1,199 @@
+import Layout from "@components/layout";
+import Header from "@components/header/header";
+import { Basicpage } from "@components/content/content";
+import { DevNote, SegmentHeading, Arrowlink } from "@components/atoms/atoms";
+import { Flex, Box, Center, Link, Heading, Text, SimpleGrid } from "@chakra-ui/react";
+import { Wallpaper, Main } from "@components/blocks/blocks";
+
+import { getAllCompanyMembers } from '@library/posts';
+
+import Hero from "@components/hero/hero";
+import { Slide, Fade } from "react-awesome-reveal";
+
+import About from "@components/about/about";
+
+import NextLink from "next/link";
+import NextImage from "next/image";
+
+// const employees = [
+//     {
+//         firstName: 'Jens',
+//         lastName: 'Reiner',
+//         picturePath: "url(images/Mitarbeiter/beispielimage-mitarbeiter.png)"
+//     },
+//     {
+//         firstName: 'Cedric',
+//         lastName: 'Döz',
+//         picturePath: "url(images/Mitarbeiter/beispielimage-mitarbeiter.png)"
+//     },
+//     {
+//         firstName: 'Jens',
+//         lastName: 'Reiner',
+//         picturePath: "url(images/Mitarbeiter/beispielimage-mitarbeiter.png)"
+//     },
+//     {
+//         firstName: 'Cedric',
+//         lastName: 'Döz',
+//         picturePath: "url(images/Mitarbeiter/beispielimage-mitarbeiter.png)"
+//     },
+//     {
+//         firstName: 'Jens',
+//         lastName: 'Reiner',
+//         picturePath: "url(images/Mitarbeiter/beispielimage-mitarbeiter.png)"
+//     },
+//     {
+//         firstName: 'Cedric',
+//         lastName: 'Döz',
+//         picturePath: "url(images/Mitarbeiter/beispielimage-mitarbeiter.png)"
+//     },
+//     {
+//         firstName: 'Jens',
+//         lastName: 'Reiner',
+//         picturePath: "url(images/Mitarbeiter/beispielimage-mitarbeiter.png)"
+//     },
+//     {
+//         firstName: 'Cedric',
+//         lastName: 'Döz',
+//         picturePath: "url(images/Mitarbeiter/beispielimage-mitarbeiter.png)"
+//     },
+//     {
+//         firstName: 'Jens',
+//         lastName: 'Reiner',
+//         picturePath: "url(images/Mitarbeiter/beispielimage-mitarbeiter.png)"
+//     },
+//     {
+//         firstName: 'Cedric',
+//         lastName: 'Döz',
+//         picturePath: "url(images/Mitarbeiter/beispielimage-mitarbeiter.png)"
+//     },
+    
+// ]
```

Comment:
If not used anymore and the provided mdx solution works you can remove this unused code